docs/build-a-runtime/index.md

```diff
@@ -209,22 +207,20 @@ Let's run the disassembler using the binary module we generated in the [Frontend
 
 `b9disasm/b9disasm ../hello.b9mod`
 
-You should now be looking at a human readable version of the Hello, World! program as represented by [base9 assembly]. You'll notice that the first three functions (`b9PrintString`, `b9PrintNumber`, and `b9PrintStack`) are the b9stdlib functions that are included in each compiled program. They can be ignored. The important part is the `b9main` function. Let's have a look at the transition between the JavaScript and the base9 assembly:
+You should now be looking at a human readable version of the Hello, World! program as represented by [base9 assembly]. You'll notice that the first three functions (`b9PrintString`, `b9PrintNumber`, and `b9PrintStack`) are the b9stdlib functions that are included in each compiled program. They can be ignored. The important part is the `<body>` function, which is the special script-body function, and the main entry to our program. Let's have a look at the transition between the JavaScript and the base9 assembly:
 
 [base9 assembly]: ../docs/Dictionary.md#base9-assembly
 
 ```js
-function b9main() {
-  b9PrintString("Hello World!");
-}
+b9PrintString("Hello World!");
 ```
 
 <figure class="image">
   <img src="{{"/assets/images/downArrow.png" | relative_url }}" width="100%"/>
 </figure>
 
 ```
-(function "b9main" 0 0
+(function "<body>" 0 0
   0  (str_push_constant 0)
   1  (function_call 0)
```

Comment:
Have we thought about a good way to get the dissembler to print other information (like function names?).  We could just add a `--verbose` that adds it in lispy style comments.

Reply:
I've been thinking about this. Right now, our printer just relies on `std::ostream` streaming of individual datums. Better printouts are going to need more context.

I'm thinking about introducing a second set of APIs where we can do a better job printing, with more information.

* `printModule(out, m)`
* `printFunction(out, m, f)`
* `printInstruction(out, m, i)`

That, and possibly contributing the sexpr printer from ab to b9, for better formatting.